<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trigger Delay</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="HeaderLabel" name="headerlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Delay</string>
   </property>
  </widget>
  <widget class="Button" name="ui0Pct">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
   <property name="clickMarginBottomSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Button" name="ui50Pct">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
   <property name="clickMarginBottomSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Button" name="ui100Pct">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="clickMarginBottomSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Slider" name="uiTriggerDelaySlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedControls">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Label" name="uiPreRecordDelay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre-Record Delay</string>
   </property>
  </widget>
  <widget class="Label" name="uiStartRecording">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>125</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Recording</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Label" name="uiEndRecording">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>125</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>End Recording</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre-Record Delay</string>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre-Trigger Recording</string>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Post-Trigger Recording</string>
   </property>
  </widget>
  <widget class="Label" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
  </widget>
  <widget class="Label" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
  </widget>
  <widget class="Label" name="label_6">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
  </widget>
  <widget class="Label" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>frames</string>
   </property>
  </widget>
  <widget class="Label" name="label_8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>frames</string>
   </property>
  </widget>
  <widget class="Label" name="label_9">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>frames</string>
   </property>
  </widget>
  <widget class="Button" name="uiDone">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="clickMarginLeftSetter" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="DecimalSpinBox" name="uiPreRecordDelaySec">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>205</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="clickMarginBottomSetter" stdset="0">
    <enum>DecimalSpinBox::none</enum>
   </property>
  </widget>
  <widget class="SpinBox" name="uiPreRecordDelayFrames">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>275</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="clickMarginTopSetter" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
  </widget>
  <widget class="DecimalSpinBox" name="uiPreTriggerRecordingSec">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>205</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="clickMarginBottomSetter" stdset="0">
    <enum>DecimalSpinBox::none</enum>
   </property>
  </widget>
  <widget class="SpinBox" name="uiPreTriggerRecordingFrames">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>275</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="clickMarginTopSetter" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
  </widget>
  <widget class="DecimalSpinBox" name="uiPostTriggerRecordingSec">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>205</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="clickMarginBottomSetter" stdset="0">
    <enum>DecimalSpinBox::none</enum>
   </property>
  </widget>
  <widget class="SpinBox" name="uiPostTriggerRecordingFrames">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>275</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="clickMarginTopSetter" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
  </widget>
  <widget class="Label" name="uiIncomingTrigger">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>64</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Incoming Trigger</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>headerlabel</zorder>
  <zorder>ui0Pct</zorder>
  <zorder>ui50Pct</zorder>
  <zorder>ui100Pct</zorder>
  <zorder>uiTriggerDelaySlider</zorder>
  <zorder>uiPreRecordDelay</zorder>
  <zorder>uiStartRecording</zorder>
  <zorder>uiEndRecording</zorder>
  <zorder>headerlabel_2</zorder>
  <zorder>headerlabel_3</zorder>
  <zorder>headerlabel_4</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>uiDone</zorder>
  <zorder>uiPreRecordDelaySec</zorder>
  <zorder>uiPreRecordDelayFrames</zorder>
  <zorder>uiPreTriggerRecordingSec</zorder>
  <zorder>uiPreTriggerRecordingFrames</zorder>
  <zorder>uiPostTriggerRecordingSec</zorder>
  <zorder>uiPostTriggerRecordingFrames</zorder>
  <zorder>uiIncomingTrigger</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>DecimalSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>decimal_spin_box</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>header_label</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label</header>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>slider</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spin_box</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiDone</tabstop>
  <tabstop>ui0Pct</tabstop>
  <tabstop>ui50Pct</tabstop>
  <tabstop>ui100Pct</tabstop>
  <tabstop>uiTriggerDelaySlider</tabstop>
  <tabstop>uiPreRecordDelaySec</tabstop>
  <tabstop>uiPreTriggerRecordingSec</tabstop>
  <tabstop>uiPostTriggerRecordingSec</tabstop>
  <tabstop>uiPreRecordDelayFrames</tabstop>
  <tabstop>uiPreTriggerRecordingFrames</tabstop>
  <tabstop>uiPostTriggerRecordingFrames</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
